tests/test_PluginLogs.py

```diff
@@ -208,7 +208,7 @@ def testLoggingPythonPlugin(self):
         pluginLogName = os.path.join(lsst.utils.getPackageDir('meas_base'), 'tests', 'plugin.log')
         directLog(log, pluginLogName)
         exposure, cat = self.dataset.realize(noise=0.0, schema=schema)
-        source = cat[0]
+        # source = cat[0]  # noqa F841: Not sure if this is deliberate
```

Comment:
I have removed them as the test passes regardless. My comment was really for the reviewer.